src/main/java/io/strimzi/kafka/bridge/http/HttpBridge.java

```diff
@@ -84,6 +90,23 @@ private void bindHttpServer(Future<Void> startFuture) {
                         startFuture.fail(httpServerAsyncResult.cause());
                     }
                 });
+
+        vertx.setPeriodic(1000, a -> {
+            log.debug("Looking for stale consumers in {} entries", timestampMap.size());
+            Iterator it = timestampMap.entrySet().iterator();
+            while (it.hasNext()) {
+                Map.Entry<String, Long> item = (Map.Entry) it.next();
+                if (item.getValue() + this.bridgeConfig.getHttpConfig().getConsumerTimeout() < System.currentTimeMillis()) {
+                    final SinkBridgeEndpoint deleteSinkEndpoint = this.httpBridgeContext.getHttpSinkEndpoints().get(item.getKey());
+                    if (deleteSinkEndpoint != null) {
+                        this.httpBridgeContext.getHttpSinkEndpoints().remove(item.getKey());
+                        deleteSinkEndpoint.close();
+                        log.warn("Consumer {} deleted after inactivity timeout ({} ms).", item.getKey(), this.bridgeConfig.getHttpConfig().getConsumerTimeout());
+                        timestampMap.remove(item.getKey());
+                    }
+                }
+            }
```

Comment:
I would move this timer set above only if the HTTP bridge is started successfully otherwise it's useless.
To avoid more code difficult to read I would factor out the timer logic in a private method as well.